package.json

```diff
@@ -29,6 +29,7 @@
     "@types/node": "^20.14.8",
     "@typescript-eslint/eslint-plugin": "^7.16.0",
     "@typescript-eslint/parser": "^7.16.0",
+    "@vue/cli-plugin-typescript": "^5.0.8",
```

Comment:
```suggestion
```

Reverting to fix build as it's not used inside the project, you may want to install with -g on your machine instead